kitchen.yml

```diff
@@ -0,0 +1,209 @@
+  - name: opensuse-leap-151-2019-2-py3
+    driver:
+      image: netmanagers/salt-2019.2-py3:opensuse-leap-15.1
+      run_command: /usr/lib/systemd/systemd
+    # Workaround to avoid intermittent failures on `opensuse-leap-15.1`:
+    # => SCP did not finish successfully (255):  (Net::SCP::Error)
+    transport:
+      max_ssh_sessions: 1
+  - name: amazonlinux-2-2019-2-py3
+    driver:
+      image: netmanagers/salt-2019.2-py3:amazonlinux-2
+  - name: ubuntu-1804-2019-2-py2
+    driver:
+      image: netmanagers/salt-2019.2-py2:ubuntu-18.04
+  - name: amazonlinux-1-2019-2-py2
+    driver:
+      image: netmanagers/salt-2019.2-py2:amazonlinux-1
+      run_command: /sbin/init
+  - name: arch-base-latest-2019-2-py2
+    driver:
+      image: netmanagers/salt-2019.2-py2:arch-base-latest
+      run_command: /usr/lib/systemd/systemd
+
+  ## SALT `2018.3`
+  - name: centos-7-2018-3-py3
+    driver:
+      image: netmanagers/salt-2018.3-py3:centos-7
+  - name: fedora-30-2018-3-py3
+    driver:
+      image: netmanagers/salt-2018.3-py3:fedora-30
+  - name: debian-9-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:debian-9
+  - name: debian-8-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:debian-8
+  - name: ubuntu-1804-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:ubuntu-18.04
+  - name: ubuntu-1604-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:ubuntu-16.04
+  - name: centos-6-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:centos-6
+      run_command: /sbin/init
+  - name: opensuse-leap-151-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:opensuse-leap-15.1
+      run_command: /usr/lib/systemd/systemd
+    # Workaround to avoid intermittent failures on `opensuse-leap-15.1`:
+    # => SCP did not finish successfully (255):  (Net::SCP::Error)
+    transport:
+      max_ssh_sessions: 1
+  - name: amazonlinux-1-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:amazonlinux-1
+      run_command: /sbin/init
+  - name: arch-base-latest-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:arch-base-latest
+      run_command: /usr/lib/systemd/systemd
+
+provisioner:
+  name: salt_solo
+  # log_level: debug
+  salt_install: none
+  require_chef: false
+  formula: pycharm
+  salt_copy_filter:
+    - .kitchen
+    - .git
+
+verifier:
+  # https://www.inspec.io/
+  name: inspec
+  sudo: true
+  # cli, documentation, html, progress, json, json-min, json-rspec, junit
+  reporter:
+    - cli
+
+suites:
+  - name: default
+    provisioner:
+      state_top:
+        base:
+          '*':
+            - pycharm.clean
+            - pycharm
+      pillars:
+        top.sls:
+          base:
+            '*':
+              - pycharm
+      pillars_from_files:
+        pycharm.sls: test/salt/default/pillar.sls
+    verifier:
+      inspec_tests:
+        - path: test/integration/default
+  - name: arch
+    provisioner:
```

Comment:
Adding inclusions and exclusions (particularly inclusions) when there are multiple suites is important, otherwise it can lead to massive lists when running `kitchen list` -- and most of these won't even work due to the nature of the suites; will be resolved when commit is added here:

https://github.com/myii/jetbrains-pycharm-formula/commit/a3bac46edc0678d4436b425e3929d5c04aea72f6#diff-fdbca9ed07a634829a3548f874dc925aR146-R168